client/dashboard/sites/overview-plan-card/index.tsx

```diff
@@ -101,15 +111,34 @@ function WpcomPlanCard( {
 			icon={ wordpress }
 			heading={ getSitePlanDisplayName( site ) }
 			description={ getCardDescription( site, purchase ) }
-			link={ site.plan?.is_free ? undefined : '/v2/me/billing/active-subscriptions' }
+			externalLink={ `/plans/${ site.slug }` }
```

Comment:
🤔 I think we want the free and paid plans to have different links

<img width="1570" height="952" alt="CleanShot 2025-08-04 at 19 49 18@2x" src="https://github.com/user-attachments/assets/41a6c41c-ed6d-49da-adb7-952e787d9a59" />

Reply:
You're right; I missed it. Fixed in https://github.com/Automattic/wp-calypso/pull/105042/commits/d9f5963f8536f842a3db222ef0ff72e2ad5ec509, thanks!